HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.Steps.cs

```diff
@@ -175,47 +176,82 @@ public void Then_only_endpoints_with_chosen_parameters_should_be_shown()
             var tag = SeleniumHelper.TestTags[0];
             var selectedTags = GetSelectedTags();
             var filteredTags = GetFilteredTags();
-            var filteredStatuses = GetFilteredStatusElements();
+            var filteredStatuses = GetFilteredStatuses();
 
             bool areStatusesCorrect = filteredStatuses.All(
-                m => m.Text.ToLower() == "faulty" || 
+                m => m.Text.ToLower() == "faulty" ||
                 m.Text.ToLower() == "healthy");
             Assert.True(areStatusesCorrect);
             Assert.True(filteredTags.Count(m => m.Text == tag) == filteredStatuses.Count);
             Assert.True(selectedTags.Any(m => string.Equals(m.Text, tag)));
         }
 
+        public void Then_endpoints_with_selected_filters_should_be_shown(int expectedCountOfSelectedTags, int expectedCountOfSelectedStatuses)
```

Comment:
This renders to: `5. THEN endpoints with selected filters should be shown [expectedCountOfSelectedTags: "1"] [expectedCountOfSelectedStatuses: "1"]`
Could you please rename method/method parameter names so the step description would be more natural?